server/models/recipe.py

```diff
@@ -0,0 +1,155 @@
+from sqlalchemy import (
+    Column, String,
+    Integer, ForeignKey,
+    Boolean, Text,
+    Float
+    )
+from sqlalchemy.orm import relationship, validates
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm.exc import NoResultFound
+from base_model import BaseModelMixin 
+
+# Base model the other model(s) will subclass
+Base = declarative_base()
+
+class Recipe(Base, BaseModelMixin):
+    """
+    Some notes
+    - chef_profile -- A relationship to chef profile related to this recipe
+    
+    - ingredients -- A list of ingredients to be converted to a string
+        then back to a list when recipe.get_ingredient_list is called
+    
+    - required_items -- A list of required utensils to be converted to a string
+        then back to a list when recipe.get_utensil_list is called
+    
+    - image_urls -- A list of image urls to be converted to a string
+        then back to a list when recipe.get_image_link_list is called
+        This will gaurentee positions in case images are ordered
+        in a certain way for a gallery as well as preventing the need
+        for saving the image as Binary in the database directly.
+    
+    - cuisine -- The category this recipe falls in.
+        **A predefined list of cuisines would be very useful here**
+    
+    - price -- The float that holds the price, to be formatted
+        to $19.00 and accessed with recipe.get_formatted_price()
+    """
+    __tablename__ = "recipe"
+
+    id = Column(Integer, primary_key=True)
+
+    # Relationships
+    chef_profile_id = Column(Integer, ForeignKey('profile.id'))
+    chef_profile = relationship('Profile')
+
+    # All fields are required hence nullable=False
+    # General Info -- No length constraints because they
+    # could all end up being fairly long
+    name = Column(String, nullable=False)
+    description = Column(Text, nullable=False)
+    available = Column(Boolean, nullable=False)
+    cuisine = Column(String, nullable=False)
+    price = Column(Integer, nullable=False)
+
+    # These store lists as strings in CSV format
+    ingredients = Column(Text, nullable=False)
+    required_items = Column(Text, nullable=False)
+    image_urls = Column(Text, nullable=False)
+
+    # TODO Validation for cuisine which 
+    # requires a predefined list of cuisines
+    # Which will also allow a select drop box to be used
+    # instead of an input box
+
+    @validates('price')
+    def validate_price(self, key, price):
+        if not isinstance(price, int):
+            raise AssertionError(f"Expected price to be of type integer but got {type(price)}")
+        new_price = price * 100
+        return new_price
```

Comment:
if you already check if it's integer, then you don't also need to multiply by 100.  
I think it'll be up to you guys whether or not you want to make that change on the frontend or the backend. If it is on the backend, I would still be a bigger fan of doing the actual conversion in the route, and use this validator to simply check if the price is an integer.